libtelemetry/src/main/java/com/mapbox/android/telemetry/Attachment.java

```diff
@@ -0,0 +1,75 @@
+package com.mapbox.android.telemetry;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Attachment extends Event implements Parcelable {
+  private static final String VIS_ATTACHMENT = "vis.attachment";
+
+  @SerializedName("event")
+  private final String event;
+  @SerializedName("attachmentMetadata")
+  private List<AttachmentMetadata> attachmentMetadata;
+  @SerializedName("files")
+  private List<byte[]> files;
```

Comment:
What are some benefits of maintaining collection of binary blobs as opposed to collection of paths to files on the filesystem? Are there any memory implications?

Reply:
No memory implications. Told that we will be receiving the data as binary from host sdk. We are working with the limitations with how the vision sdk currently works.